components/Impact/PersonCard.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import styles from '../../styles/pages/impact.module.scss';
+
+export default function PersonCard({name, pronouns, position, major, year, img, alt}){
```

Comment:
Not sure how much @mattxwang has been able to talk about this, but we've got quite a few "components that feature 1 person and their info".

Would be curious what y'all think about creating a unified version of these `PersonCard` components. Tagging in https://github.com/uclaacm/website/issues/264

Reply:
Yeah, if you don't mind let's just wrap this into #264 - eager to get these changes somewhat off my plate + it makes for easier deployment for now.